packages/manager/apps/dedicated/client/app/dedicated/server/details/translations/Messages_fr_FR.json

```diff
@@ -1273,11 +1273,11 @@
   "server_error_bandwidth_order": "Une erreur est survenue lors de la commande de bande passante : {{error}}",
   "server_configuration_task_INFRA_002_VIRTUAL_NETWORK_INTERFACE_GROUP": "Interfaces réseau de groupe",
   "server_configuration_task_INFRA_002_VIRTUAL_NETWORK_INTERFACE_UNGROUP": "Dissocier les interfaces réseau",
-  "server_configuration_no_intervention": "Interventions sur site",
+  "server_configuration_no_intervention": "Désactiver les interventions sur site",
   "server_configuration_on": "On",
   "server_configuration_off": "Off",
-  "server_configuration_no_intervention_activate_success": "L'intervention sur site est activée sur le serveur {{serverName}}",
+  "server_configuration_no_intervention_activate_success": "L'intervention sur site est désactivée sur le serveur {{serverName}}",
   "server_configuration_no_intervention_activate_failed": "Une erreur est survenue lors de l'activation des interventions sur site sur le serveur {{serverName}}",
-  "server_configuration_no_intervention_deactivate_success": "L'intervention sur site est désactivé sur le serveur {{serverName}}",
+  "server_configuration_no_intervention_deactivate_success": "L'intervention sur site est activée sur le serveur {{serverName}}",
```

Comment:
Maybe it should be easier to invert translations displayed in HTML ? (As also it doesn't make sense to have a key `deactivate` to say something has been activated imo)

Reply:
With my current understanding, the keys and messages are not inverted. I might be wrong as I dont understand the content. I am using the same content as provided by the requester. Visiting the UI might give better understanding about it. happy to change if required.

Reply:
The key specifies 'deactivate' but the content says something is activated :)